tq/api.go

```diff
@@ -38,6 +41,7 @@ func Batch(m *Manifest, dir Direction, remote string, remoteRef *git.Ref, object
 	}
 
 	return m.batchClient().Batch(remote, &batchRequest{
+		HashAlgorithm:        "sha256",
```

Comment:
In the `type batchRequest struct` definition just above, `HashAlgorithm` is the last element, while here it's given as the first.  It makes no practical difference but as the others are in the same order, perhaps this should also go last?

Reply:
Fixed.